command/s3_put.go

```diff
@@ -71,11 +71,12 @@ type s3put struct {
 	// Optional, when set to true, causes this command to be skipped over without an error when
 	// the path specified in local_file does not exist. Defaults to false, which triggers errors
 	// for missing files.
-	Optional bool `mapstructure:"optional"`
+	Optional util.StringOrBool `mapstructure:"optional" plugin:"expand"`
```

Comment:
What does the plugin tag do/where is it used?

Reply:
it's the thing that makes expansions work